test/SlowTests/Issues/RavenDB-14541.cs

```diff
@@ -0,0 +1,247 @@
+                }
+
+                using (var session = store.OpenSession())
+                {
+                    var query3 = from u in session.Query<Order>()
+                        let includes = RavenQuery.Include<Order>(u.Employee)
+                        select new QueryResult { Comapny = u.Company };
+
+                    var results = query3.ToList();
+                }
+            }
+        }
+        catch (Exception e)
+        {
+            Assert.NotNull(e);
+            Assert.Contains("You can't use the include that way try: let _ = RavenQuery.Include<T>(Expression<Func<T, string>> path)", e.InnerException.Message);
+        }
+    }
+
+    [Fact]
+    public void ShouldThrowNotSupportedExceptionHaveIncorrectObjectAsArg()
+    {
+        try
+        {
+            using (DocumentStore store = GetDocumentStore())
+            {
+                using (var session = store.OpenSession())
+                { 
+                    session.Store(new Address { CountryState = "states/1#zip07", City = "new-york", StateId = "states/1" });
+                    session.Store(new Address { CountryState = "states/2#zip05", City = "haifa", StateId = "states/2" });
+
+                    session.Store(new State { Name = "Alabama" }, "states/1");
+                    session.Store(new State { Name = "Minassota" }, "states/2");
+
+                    session.SaveChanges();
+                }
+                using (var session = store.OpenSession())
+                {
+                    var query3 = from a in session.Query<Address>()
+                        let _ = RavenQuery.Include<Address>(a.StateId)
+                        select new { Name = a.City };
+
+                    var res = query3.ToList();
+                }
+            }
+        }
+        catch (Exception e)
+        {
+            Assert.NotNull(e);
+            Assert.Contains("You can't use the include that way try: let _ = RavenQuery.Include<T>(Expression<Func<T, string>> path)", e.InnerException.Message);
+        }
+    }
+
+    [Fact]
+    public void SessionQuerySelectAddressFromIncludeDoc_UsingRavenQueryStringWithStateObject()
+    {
+        using (DocumentStore store = GetDocumentStore())
+        {
+            using (var session = store.OpenSession())
+            {
+                session.Store(new Address { CountryState = "states/1#zip07", City = "new-york", StateId= "states/1" });
+                session.Store(new Address { CountryState = "states/2#zip05", City = "haifa", StateId = "states/2" });
+
+                session.Store(new State { Name = "Alabama" }, "states/1");
+                session.Store(new State { Name = "Minassota" }, "states/2");
+
+                session.SaveChanges();
+            }
+
+            using (var session = store.OpenSession())
+            {
+                var query3 = from a in session.Query<Address>()
+                    let _ = RavenQuery.Include<State>("a.StateId")
```

Comment:
should be `let _ = RavenQuery.Include<State>("StateId")`

Reply:
done